packages/maskbook/src/extension/background-script/EthereumServices/send.ts

```diff
@@ -54,22 +51,26 @@ function getChainIdFromPayload(payload: JsonRpcPayload) {
     }
 }
 
-function isRpcNeedToBeConfirmed(payload: JsonRpcPayload) {
-    return [
-        EthereumMethodType.ETH_SIGN,
-        EthereumMethodType.PERSONAL_SIGN,
-        EthereumMethodType.ETH_SIGN_TYPED_DATA,
-        EthereumMethodType.ETH_DECRYPT,
-        EthereumMethodType.ETH_GET_ENCRYPTION_PUBLIC_KEY,
-        EthereumMethodType.ETH_SEND_TRANSACTION,
-    ].includes(payload.method as EthereumMethodType)
+function handleRecentTransaction(account: string, response: JsonRpcResponse | undefined) {
+    const hash = response?.result as string | undefined
+    if (typeof hash !== 'string') return
+    if (!/^0x([\dA-Fa-f]{64})$/.test(hash)) return
```

Comment:
extract the regexp pattern to `@masknet/shared`?

<img width="519" alt="CleanShot 2021-09-03 at 06 59 06@2x" src="https://user-images.githubusercontent.com/3842474/131926656-01829118-8758-47bd-a5b0-eb72855e6a05.png">

Reply:
or use `@masknet/web3-shared`?